src/helpers/index.js

```diff
@@ -54,7 +54,7 @@ export default {
 		}
 		var filtered = [];
 
-		coreHelpers.each(collection, function(item) {
+		collection.forEach(function(item) {
```

Comment:
`collection` is not an array or array does not have `filter` here. Are we sure `where` is not used to some other type of collection (its public)?

Reply:
I've gone ahead and removed `where` as well